docs/pages/enterprise/workflow/ssh-approval-mattermost.mdx

```diff
@@ -4,9 +4,7 @@ description: This guide explains how to setup a Mattermost plugin for Teleport f
 h1: Teleport Mattermost Plugin Setup
 ---
 
-This guide will talk through how to setup Teleport with Mattermost. Teleport to
-Mattermost integration allows teams to approve or deny Teleport access requests
-using [Mattermost](https://mattermost.com/) an open source messaging platform.
+This guide will talk through how to set up Teleport with Mattermost. Teleport's Mattermost integration allows teams to approve or deny Teleport access requests using [Mattermost](https://mattermost.com/), an open source messaging platform.
```

Comment:
Feels like the link should be on the first mention of Mattermost, not the second.